src/ParamPassing.cc

```diff
@@ -148,6 +184,13 @@ ElementPtr getElementById(const SDFPtr _sdf,
       childElem = matchingChild->GetFirstElement();
       matchingChild = nullptr;
       startIdx = longestIdx;
+
+      size_t found = _elemId.substr(startIdx).find("::");
```

Comment:
nit: Wouldn't it be more efficient to do `_elemId.find("::", startIdx)`?

Reply:
Done fb0bf0de4eb1e8742345fb5c84483db7be110ec4